monkey/infection_monkey/utils/monkey_log_path.py

```diff
@@ -1,20 +1,39 @@
 import os
+import string
 import sys
+import time
+from random import SystemRandom
 
-from infection_monkey.config import WormConfiguration
+LOG_DIR_LINUX = "/tmp/"
+LOG_DIR_WINDOWS = "%temp%\\"
```

Comment:
If we specify these, there may be systems that we can not run on. If we use `mkstemp`, it will find the appropriate temporary directory and put the files in it. You just have to read about where mkstemp puts files ([gettempdir()](https://github.com/guardicore/monkey/issues/1761)) and update the documentation accordingly.